brackend/tasks/auth.py

```diff
@@ -1,46 +1,39 @@
-import datetime
+from functools import wraps
 
-import bcrypt
-import jwt
-from firebase_admin.auth import create_user, generate_email_verification_link
+from firebase_admin.auth import verify_id_token
+from flask import request
+from flask_restful import abort
 
-from brackend.util import SECRET_KEY, BrackendException
 
-EXPIRY_DAYS = 14
+def abrt():
+    """Abort with a 401 response code."""
+    abort(401, message="Authorization Required")
 
 
-class AuthenticationError(BrackendException):
-    pass
+def auth_decorator(meth):
```

Comment:
Can we call this something more like `require_auth` - we should know its a decorator from the @wraps